tasks/build.yml

```diff
@@ -3,12 +3,12 @@
 - name: Assert vendor variable
   assert:
     that:
-      - java.vendor in [ 'oracle', 'openjdk' ]
+      - sansible_java_vendor in [ 'oracle', 'openjdk' ]
 
 - name: Build Oracle Java
   include: build/build-oracle.yml
-  when: java.vendor == 'oracle'
+  when: "sansible_java_vendor == 'oracle'"
```

Comment:
I think when clauses should be left bare (ie. unquoted) I might be wrong on that though.

Reply:
Seen both and I think `"'foo' in bar"` requires them, but I'm not fighting this one :-)